charts/scaffold/templates/copy-secrets-job.yaml

```diff
@@ -0,0 +1,64 @@
+{{- if .Values.copySecretJob.enabled }}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  namespace: {{ .Values.tuf.forceNamespace }}
```

Comment:
For the `namespace` field, take a look how the other charts leverage a partial called `<chart>.namespace`. This capability is one of the components that will be added to a common library chart in the future

Reply:
Yes, this is an interesting one. Adressed for TUF in c5c116d.
If we want to do it properly, we should use a similar macro here for all the other namespaces where the secret is copied from, like rekor, fulcio and ctlog.